runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -2057,7 +2066,7 @@ static void compute_comm_dom_cnt(void)
     if (comm_dom_cnt == 0) {
       uint32_t num_PUs;
 
-      if ((num_PUs = chpl_getNumLogicalCpus(true)) > 0)
+      if ((num_PUs = chpl_task_getMaxPar()) > 0)
```

Comment:
Also, in the past I have tried to call `chpl_task_getMaxPar()` before the tasking layer was initialized (in an early version of https://github.com/chapel-lang/chapel/pull/6893 -- but that was from `chpl_comm_init()`, not `chpl_comm_post_task_init()`. 

Because of my previous screw-up I wanted to check to make sure this call was after the tasking layer was initialized (and indeed itis). I wonder if we should add a comment above the `compute_comm_dom_cnt()` declaration that says "Note, must be called after the tasking layer is initialized" just to ensure we don't move it too early in the future? (I'm a little torn on that because that comment will likely become out of date)

Reply:
I feel like it's okay to leave this without a comment, since by its name we know `chpl_comm_post_task_init()` is called after tasking is running, and there's no reason to need the comm domain count before where it now is computed.  (Could be "famous last words", I guess.)

Reply:
Oh, hmm, you were talking about adding a comment before the `compute_comm_dom_cnt()` decl, not the call.  Yeah, probably a good idea.  I'll do that.